src/eval.c

```diff
@@ -90,24 +149,37 @@ RESOLVE_CALLS_FUNC_SIGNATURE(resolve_calls_call) {
 						   symbol_info->ast_node_id);
 			}
 		}
+
+		size_t new_node = VEC_AT(&VEC_AT(&ast->nodes, ast_node_id).childs, 1);
+
+		resolve_symbols(ctx, ast, curr_scope_id, new_node);
 	} else {
 		fprintf(stderr, "ERROR: No symbol with name \"%s\" exist\n", call_name);
 	}
+
+	const struct ast_node *node = &VEC_AT(&ast->nodes, ast_node_id);
+
+	for (size_t i = 0; i < node->childs.len; i++) {
+		resolve_calls(ctx, ast, curr_scope_id, VEC_AT(&node->childs, i));
+	}
```

Comment:
We already have a function to do this.
```suggestion
	resolve_calls_program(ctx, ast, curr_scope_id, ast_node_id);
```